### PR TITLE
Feature/caes 368  fix envs

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -22,7 +21,7 @@ services:
         depends_on:
             - php
         ports:
-            - "${SERVER_HTTP_PORT}:80"
+            - "80:80"
```
Comment (beshkenadze) [inline docker-compose.yml]: Please don't use the default ports. Use an environment variable to set the web port — is correct.

Comment (mg-4xxi) [reply docker-compose.yml]: done

File: .env.dist [hunk 2]
```diff
@@ -1,70 +1,74 @@
...
-REDIS_PORT=6379
-###< snc/redis-bundle ###
-###> rabbitMQ ###
-RABBITMQ_HOST=rabbitmq
-RABBITMQ_VHOST=/
-RABBITMQ_DEFAULT_USER=rabbitmq
-RABBITMQ_DEFAULT_PASS=rabbitmq
-###< rabbitMQ ###
\ No newline at end of file
+#### RabbitMQ version 3.7.14
```
Comment (atrutko-4xxi) [inline .env.dist]: Is RabbitMQ service optional? @mg-4xxi

Comment (mg-4xxi) [reply .env.dist]: just already has default values
this is important for security but not important to deploy the project

Comment (atrutko-4xxi) [reply .env.dist]: I do not agree. Without this service, sending mail will not work for you, and therefore, this service is just as important as a database.

Comment (mg-4xxi) [reply .env.dist]: replaced

**Pull request merged**